contracts/dapis/DapiServer.sol

```diff
@@ -495,6 +495,40 @@ contract DapiServer is
         emit UpdatedBeaconWithSignedData(beaconId, decodedData, timestamp);
     }
 
+    /// @notice Updates a Beacon using data signed for this contract by the
+    /// respective Airnode, without requiring a request or subscription
+    /// @param airnode Airnode address
+    /// @param templateId Template ID
+    /// @param timestamp Timestamp used in the signature
+    /// @param data Response data (a `uint256` encoded in contract ABI)
+    /// @param signature Template ID, a timestamp and the response data signed
+    /// for this contract by the Airnode address
```

Comment:
No need to mention that the `chainId` and the `contract address` is also part of the signed payload?

Reply:
That's what I meant by "signed for this contract"